pages/common/arping.md

```diff
@@ -0,0 +1,28 @@
+# arping
+
+> Discover and probe hosts in a network using the ARP protocol.
+> Useful for MAC addres discovery.
+
+- Ping a host by ARP request packets:
+
+`arping {{host_ip}}`
+
+- Ping a host on a specific interface:
+
+`arping -I {{interface}} {{host_ip}}`
+
+- Ping a host and stop at the first reply:
+
+`arping -f {{host_ip}}`
+
+- Ping a host a specific number of times:
+
+`arping -c {{count}} {{host_ip}}`
+
+- Broadcast ARP request packets to update neighbours' ARP caches:
```

Comment:
Is the apostrophe really necessary in this case?

Reply:
Well, yes, it's a possessive apostrophe.

Reply:
Understood. 

No other comments from my side.